pvp/controller/control_module.py

```diff
@@ -988,7 +1016,7 @@ def set_flow_out(self, Qout, dt):
         Qout_clip = np.min([Qout, 2])                    # Flows have to be positive, and reasonable. Nothing here is faster that 2 l/s
         Qout_clip = np.max([Qout_clip, 0])
         difference_pressure = self.current_pressure - 0  # Convention: outside is "0"
-        conductance = 0.05*Qout_clip                     # This should be in the range of ~1 liter/s for typical max pressure differences
+        conductance = 0.01*Qout_clip                     # This should be in the range of ~1 liter/s for typical max pressure differences
```

Comment:
makes the simulation a neater; it simulates a narrower constriction, so that flow-out is slower